apps/paypal-provider/src/pages/payment/[intent].tsx

```diff
@@ -0,0 +1,73 @@
+import { PayPalButtons, PayPalScriptProvider } from '@paypal/react-paypal-js';
+import React from 'react';
+import config from '../../config/constants';
+import getClientId from '../../graphql/backend/getClientId';
+
+export interface HomeProps {
+  clientId: string;
+  apiKey: string;
+  intent: string;
+}
+
+const Home: React.FunctionComponent<HomeProps> = ({ clientId, apiKey, intent }) => {
+  //TOOD: style, show data....
+
+  return (
+    <div>
+      <PayPalScriptProvider
+        options={{
+          'client-id': clientId,
+        }}
+      >
+        <PayPalButtons
+          style={{ layout: 'horizontal', color: 'blue', tagline: false }}
```

Comment:
Is it possible to use className instead of style :smile:

Reply:
it's possible to add className if needed
but I think it's better to use style here since it's not classic style prop but a customized one:
feel free to  change this page :D 

(JSX attribute) PayPalButtonsComponentOptions.style?: {
    color?: "blue" | "gold" | "silver" | "white" | "black";
    height?: number;
    label?: "paypal" | "checkout" | "buynow" | "pay" | "installment" | "subscribe" | "donate";
    layout?: "horizontal" | "vertical";
    shape?: "rect" | "pill";
    tagline?: boolean;
}